### DISCUSSION TITLE
Visibility of diff

### DISCUSSION THREAD

**Discussion by veryphatic** [tractr/directus-sync#59] [General]
Hi there, I'm wondering if there is a way to be able to see the diff changes, rather than just a summary? I built in Directus diff is able to provide this detail. For me it'd be useful to have visibility as part of a CI process

**Comment by EdouardDem** [comment]
Hi. Currently, it is not possible to change the diff output. However, this could be a great feature. Are you thinking of a diff similar to what Git or Terraform use? I also have a table output in mind.

**Comment by veryphatic** [comment]
I was thinking something similar to the current Directus diff output, but something like Git would be nice 😊

**Reply by EdouardDem** [reply]
Sorry, I did get the "Directus Diff." I understand your point now. For example: https://docs.directus.io/reference/system/schema.html#example-2
I will add this to the backlog.
